app/datastore/pg/users.go

```diff
@@ -178,14 +179,14 @@ func (u *userRepo) UpdateEventById(ctx context.Context, eventId int, updatedEven
 	statement := `
 		UPDATE events
 		SET
-			event_name = $2,
-			event_date = $3,
-			event_location = $4,
-			uuid = $5
+			event_name = $3,
+			event_date = $4,
+			event_location = $5,
+			uuid = $2
 		WHERE id = $1
 	`
 
-	_, err := u.dbInstance.ExecContext(ctx, statement, eventId, updatedEvent.EventName, updatedEvent.EventDate, updatedEvent.EventLocation, updatedEvent.Uuid)
+	_, err := u.dbInstance.ExecContext(ctx, statement, eventId, updatedEvent.Uuid, updatedEvent.EventName, updatedEvent.EventDate, updatedEvent.EventLocation)
```

Comment:
go ahead and order the sql statement too, so that the top line is $2 and the last line is $5

Reply:
its above the lines i ref'd

Reply:
ok